src/common/access_pattern.h

```diff
@@ -20,7 +20,8 @@
 /** A single seek entry */
 struct seek_entries {
 	int32_t operation; /**< read or write */
-	int32_t reqsize; /**< Size of data transfer in blocks */
+	int32_t blocksize; /**< Size of data transfer in blocks */
+	int32_t reqsize; /**< Number of blocks in the request */
```

Comment:
Why are these signed? Are negative values expected?

Reply:
When I looked at the code, it was already use `int32_t`